### PR TITLE
Distinguish between x64 and x86 installer

### PR DESCRIPTION
**Pull request by SinghRajenM**
Distinguish between x64 and x86 installer in control panel

installer_x86_x64

### CODE REVIEW

**Comment by MAPJe71** [comment]
To be honest I prefer a `(64-bit)` and a `(32-bit)` suffix respectively (there's no need to follow MS)

File: PowerEditor/installer/nsisInclude/tools.nsh [hunk 1]
```diff
@@ -128,7 +128,11 @@ Function writeInstallInfoInRegistry
 	WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\notepad++.exe" "" "$INSTDIR\notepad++.exe"
 	
 	WriteRegStr HKLM "Software\${APPNAME}" "" "$INSTDIR"
-	WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "DisplayName" "${APPNAME}"
+	!ifdef ARCH64
+		WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "DisplayName" "${APPNAME} (x64)"
```
Comment (donho) [inline PowerEditor/installer/nsisInclude/tools.nsh]: it's rather "(64-bit x64)"

File: PowerEditor/installer/nsisInclude/tools.nsh [hunk 2]
```diff
@@ -128,7 +128,11 @@ Function writeInstallInfoInRegistry
...
+	!else
+		WriteRegStr HKLM "${UNINSTALL_REG_KEY}" "DisplayName" "${APPNAME} (x86)"
```
Comment (donho) [inline PowerEditor/installer/nsisInclude/tools.nsh]: It's rather (32-bit x86)

**Comment by SinghRajenM** [comment]
@MAPJe71 x64 and x86 is no more related to MS. I just took reference to show the difference. Many of applications such as Firefox (can see in the screenshot as well), 7zip etc use it.

Anyway, the intention was to differentiate the npp version in installed program list.
I'm sure, lot of DV PCs are loaded with 32bit NPP version. After installing x64 bit, if they wish to uninstall 64bit (may be due to lack of plugins at this moment) or uninstall 32bit (they may like 64bit more), then it is hard to identify which one to pick. 
However, from uninstaller window (_if they really look at Uninstalling from C:\Program Files **(x86)**\Notepad++_), they may realize which version they are going to uninstall.

So just to simplify this task, it is good to write 64bit or 32bit details as well.

File: PowerEditor/installer/nsisInclude/tools.nsh (hunk 2 above)
Comment (SinghRajenM) [inline PowerEditor/installer/nsisInclude/tools.nsh]: OK, I will update as "32-bit x86" and "64-bit x64".

**Pull request closed without merging**